chemprop/uncertainty/uncertainty_calibrator.py

```diff
@@ -340,6 +363,7 @@ def nll(
             pred_bins = np.searchsorted(bin_edges, task_abs_z)
             # magnitude adjusted by stdev scale of the distribution and symmetry assumption
             task_likelihood = bin_magnitudes[pred_bins] / task_stdev / 2
+            task_likelihood = np.where(task_mask, task_likelihood, 0.)
             nll[:, i] = -1 * np.log(task_likelihood)
         return nll
```

Comment:
You're right, that was sloppy of me. I should  have done as you suggested here. In the methods that I wrote later though, I masked things differently and just removed them from the task_preds and task_targets matrices. I will do that here because I think it's cleaner than the replacement with where.